edgexfoundry/edgex-go
# feat(snap): add additional tokens for app-service-configurable profiles

The external-mqtt-trigger and app-push-to-core profiles are used by the TAF integration tests.

This PR enables those profiles by creating tokens for them and making the tokens available for
the edgex-app-service-configurable snap.

Signed-off-by: Siggi Skulason <siggi.skulason@canonical.com>







**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

  <link to docs PR>

## Testing Instructions


1. Create a clean build of the snap 
2. Install the snap
3. Confirm the token exists: `sudo ls /var/snap/edgexfoundry/current/secrets/app-external-mqtt-trigger/secrets-token.json`

## New Dependency Instructions (If applicable)
